.github/workflows/docs.yml

```diff
@@ -0,0 +1,39 @@
+name: Build docs
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Install dependencies
+        run: pip install poetry sphinx sphinx-rtd-theme
+
+      - name: Install all pip packages
+        run: |
+          cd ./frontend/ && poetry export -f requirements.txt > ../requirements.txt
+          cd ../backend/ && poetry export -f requirements.txt >> ../requirements.txt
+          pip install -r ../requirements.txt
```

Comment:
Hmm... I do not see any other way of doing this, but with this approach what happens if the same dependency is used in both backend and frontend, will there be some sort of conflict? The same dependency would appear twice in the requrements.txt, theoretically possibly with different versions.

Reply:
Pretty sure it crashes, it was the only solution I came up with at the time but it might work to modify the `generateDocs` script to assume that there is a `.venv/` available in the frontend and backend folders. I'll try it later today to see if it works.